src/components/Layout.js

```diff
@@ -27,7 +27,6 @@ const Main = styled.main`
 const Layout = ({ children }) => {
   return (
     <>
-      <MetaHead />
```

Comment:
This needs to move to `_app.js` otherwise pages lower in the page tree won't be able to overwrite meta tags.